src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -216,14 +314,37 @@ public IsNullFilter(String path) {
    */
   public static class ExistsFilter extends SetFilterBase<String> {
     public ExistsFilter(String path, boolean existFlag) {
-      super("exist_keys", path, Operator.CONTAINS);
+      super("exist_keys", path, path, Operator.CONTAINS);
+    }
+
+    @Override
+    JsonNode asJson(JsonNodeFactory nodeFactory) {
+      return null;
+    }
+
+    @Override
+    boolean canAddField() {
+      return false;
```

Comment:
I guess this is tied to above, where "field $eq null" should insert `null` value on creation or not.

Reply:
This is $exists filter, we won't be adding data data to new row. So canAddField() return false.

Reply:
Ah ok.